src/PrestaShopBundle/Resources/views/Admin/Improve/Payment/PaymentMethods/payment_methods.html.twig

```diff
@@ -45,5 +62,6 @@
         {% endif %}
       </div>
     </div>
+
```

Comment:
Please remove this empty added line

Reply:
But I like it like this 🎶

Reply:
Removing it in order to please you so you approve my PR 😄(but I really find the code easier to read with this blank line as a separator)